lib/ukrand/chacha.c

```diff
@@ -111,7 +111,26 @@ void uk_swrand_init_r(struct uk_swrand *r, unsigned int seedc,
 	r->k = 16;
 }
 
-__u32 uk_swrand_randr_r(struct uk_swrand *r)
+static int _uk_chacha_init(struct uk_init_ctx *ictx __unused)
+{
+	unsigned int seedc = 10;
+	__u32 seedv[10];
+	unsigned int i;
+
+	uk_pr_info("Initialize random number generator...\n");
+
+	for (i = 0; i < seedc; i++) {
+		int ret = ukarch_random_seed_u32(&seedv[i]);
```

Comment:
Define this above, together with the other local variables. You can ignore Codacy's warnings about reduced scopes.